apache-ansible-demo/docs/content.md

```diff
@@ -0,0 +1,13 @@
+The resource package contains resource definitions and scripts that allow to manage an Apache server on OpenStack through the Ansible Lifecycle Driver.
+
+The package contains the descriptor of an assembly ([apache-ansible-demo](./Descriptor/assembly.yml)) and of a resource ([hw-apache](./Contains/hw-apache-vnfc/Definitions/lm/resource.yaml)).  
```

Comment:
The links to the resource and assembly descriptor are incorrect. You need to go up one level e.g. (../Descriptor/assembly.yml)